api/eds.proto

```diff
@@ -172,21 +172,32 @@ message UpstreamLocalityStats {
   // originate at the client and the endpoint responded successfuly. For gRPC,
   // the grpc-status values are those not covered by total_error_requests below.
   uint64 total_successful_requests = 2;
+
   // The total number of unfinished requests
   uint64 total_requests_in_progress = 3;
+
   // The total number of requests that failed due to errors at the endpoint.
   // For HTTP these are responses with 5xx status codes and for gRPC the
-  // grpc-status values {DeadlineExceeded, Unimplemented, Internal,
-  // Unavailable, Unknown, DataLoss}.
+  // grpc-status values:
+  //
+  //   - DeadlineExceeded
+  //   - Unimplemented
+  //   - Internal
+  //   - Unavailable
+  //   - Unknown
+  //   - DataLoss
   uint64 total_error_requests = 4;
 
   // Stats for multi-dimensional load balancing.
   repeated EndpointLoadMetricStats load_metric_stats = 5;
 }
 
-// Per cluster stats
+// Per cluster load stats. Envoy reports these stats a management server in a
+// :ref:`LoadStatsRequest<envoy_api_msg_LoadStatsRequest>`
 message ClusterStats {
+  // The name of the cluster.
   string cluster_name = 1;
+
   // Need at least one.
   repeated UpstreamLocalityStats upstream_locality_stats = 2;
```

Comment:
Use constraints here for >= 1 (and change wording so it's not redundant with the constraint).